pennylane_lightning_kokkos/src/tests/CMakeLists.txt

```diff
@@ -36,9 +36,10 @@ target_link_libraries(runner_kokkos INTERFACE OpenMP::OpenMP_CXX)
 endif()
 
 target_link_libraries(runner_kokkos PUBLIC pennylane_lightning_kokkos Catch2::Catch2 kokkos)
-target_sources(runner_kokkos PRIVATE Test_StateVectorKokkos_Measure.cpp Test_StateVectorKokkos_NonParam.cpp Test_StateVectorKokkos_Param.cpp Test_StateVectorKokkos_Generator.cpp Test_StateVectorKokkos_Expval.cpp Test_AdjointDiffKokkos.cpp)
+target_sources(runner_kokkos PRIVATE Test_Backend_info.cpp Test_StateVectorKokkos_Measure.cpp Test_StateVectorKokkos_NonParam.cpp Test_StateVectorKokkos_Param.cpp Test_StateVectorKokkos_Generator.cpp Test_StateVectorKokkos_Expval.cpp Test_AdjointDiffKokkos.cpp)
 
 target_include_directories(runner_kokkos INTERFACE ${kokkos_INC_DIR})
+target_include_directories(runner_kokkos PUBLIC "../bindings/")
```

Comment:
If you need those header defined bindings functions made earlier in a different file, best to move them elsewhere into a separate translation unit (.hpp, cpp) and link as normally with a CMake target. We shouldn't be using relative paths like this as they are unreliable.